### PR TITLE
✨Source Stripe: New Stream - Persons

### LINKED ISSUE #16965: Source Stripe: new stream Person
## Tell us about the problem you're trying to solve
[Person](https://stripe.com/docs/api/persons)
This is an object representing a person associated with a Stripe account.
A platform cannot access a Standard or Express account's persons after the account starts onboarding, such as after generating an account link for the account. See the [Standard onboarding](https://stripe.com/docs/connect/standard-accounts) or [Express onboarding documentation](https://stripe.com/docs/connect/express-accounts) for information about platform pre-filling and account onboarding steps.

[Permission](https://stripe.com/docs/security/permissions)

## Describe the solution you’d like
Research pros and cons for implementing this stream.
_Research should contain: what stream should be implemented, permissions and scopes (if needed), paid plan (if needed)._

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #16965

## Enhancement
New streams: 

- Persons

## Recommended reading order
1. `source_stripe/source.py`
2. `source_stripe/streams.py`
3.  `source_stripe/schemas/*`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed
+

### CODE REVIEW

File: airbyte-integrations/connectors/source-stripe/source_stripe/streams.py [hunk 1]
```diff
@@ -705,3 +705,22 @@ class Accounts(StripeStream):
 
     def path(self, **kwargs):
         return "accounts"
+
+class Persons(IncrementalStripeStream):
+    """
+    API docs: https://stripe.com/docs/api/persons/list
+    """
+    name = "persons"
+
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-stripe/source_stripe/streams.py:714]: Need to declare `cusor_field = "created"` here.

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+      }
+    },
+    "created": {
+      "type": "number"
+    },
+    "dob": {
+      "type": ["null", "object"],
+      "properties": {
+        "day": {
+          "type": ["null", "string"]
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:46]: These properties are integers per https://stripe.com/docs/api/persons/object#person_object-dob-day

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+        },
+        "title": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "requirements": {
+      "type": ["null", "object"],
+      "properties": {
+        "alternatives": {
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:206]: The properties of this object doesn't align with what stripe says the structure should be. https://stripe.com/docs/api/persons/object#person_object-requirements-alternatives

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json [hunk 4]
```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Persons Schema",
+  "additionalProperties": true,
+  "type": ["null", "object"],
+  "properties": {
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:6]: It looks like we are missing some properties. If we can add the following just to ensure full compatibility: `phone, email, address_kana, address_kanji, first_name_kana, first_name_kanji, full_name_aliases, gender, id_number_secondary_provided, maiden_name, nationality, political_exposure, registered_address`

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+        "postal_code": {
+          "type": ["null", "string"]
+        },
+        "state": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "created": {
+      "type": "number"
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:40]:
```suggestion
      "type": ["null", "integer"]
```

**Review by archangelic — CHANGES_REQUESTED** [review]
Thanks for your contribution! I have added some suggested changes. Let me know if there are any questions.

**Comment by marcosmarxm** [comment]
@archangelic can we get an update in this contribution?

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json (hunk 2 above)
Comment (archangelic) [reply airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:82]: `day`, `month`, and `year` need to be set to integers.

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json (hunk 3 above)
Comment (archangelic) [reply airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json:242]: According to the docs, `alternatives` should have 2 properties `alternative_fields_due` and `original_fields_due` that are arrays of strings.

**Comment by archangelic** [comment]
Looks like there just needs two areas with changes, then we can continue with getting this merged!

**Review by midavadim — APPROVED** [review]

**Comment by midavadim** [comment]
Related PR with CI tests:
https://github.com/airbytehq/airbyte/pull/28322

**Pull request merged**